dali/python/nvidia/dali/ops/__init__.py

```diff
@@ -581,8 +599,11 @@ def __call__(self, *inputs, **kwargs):
             return result
 
     Operator.__name__ = str(name)
-    Operator.schema_name = schema_name or Operator.__name__
-    Operator._generated = True  # The class was generated using python_op_factory
+    Operator.schema_name = schema_name
+    Operator._internal_schema_name = internal_schema_name
+    # The class was generated using python_op_factory, and we don't expect custom wrapper.
+    # If needed, allow this tag to be overridden by an argument to this function
+    Operator._generated = internal_schema_name is None
```

Comment:
1. What I've written at the declaration.
2. Now that this attribute is always there (?) we should probably get rid of those `getattr(op, "_generated", None)`

Reply:
This still leaves the External Source without the use of python_op_factory, maybe I can look at it as a follow-up, but the ExternalSourceGroup captures the OperatorInstances directly, so this whole abstraction is not usable there at this point in time.